guide/1-Essentials/custom-tags.md

```diff
@@ -79,6 +79,7 @@ let right = <Right>
 Custom tags can inherit from other custom tags, or from native tags. E.g. if you want to create a custom form component, you can simply inherit from form:
 
 ```imba
+# define a custom tag, inheriting from Form
 tag RegisterForm < Form
```

Comment:
I think this is a typo, `Form` is not define anywhere here. Can you change it to lower case `form`?
```
# define a custom tag, inheriting from form
tag RegisterForm < form
```

Reply:
Done